src/cli/README.md

```diff
@@ -114,7 +114,7 @@ Done
 
 ### extpanid \<extpanid\>
 
-Set the Thread Extended PAN ID value.
```

Comment:
I don't have strong feelings about this, but, to the extent that we are directly referencing something that is a Thread concept that is not specific to OpenThread, I would tend to think we the use of the term *Thread* is appropriate.